cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,93 @@
+"""Main Entry point for cloud function."""
+import os
+
+from google.cloud import firestore
+from firebase_admin import auth, initialize_app
+from firebase_functions import firestore_fn, https_fn, options
```

Ruff (0.8.2)

`firebase_functions.firestore_fn` imported but unused

Remove unused import

(F401)